src/containers/ImportResources/ImportResources.js

```diff
@@ -0,0 +1,221 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+
+// import Link from 'react-router-dom';
+import {
+  ToastNotification,
+  TextInput,
+  FormLabel,
+  Button,
+  Tooltip
+} from 'carbon-components-react';
+import { connect } from 'react-redux';
+
+import '../../components/Definitions/Definitions.scss';
```

Comment:
Components shouldn't be loading styles directly from other components.

If we need to extract common styles or rewrite things to apply globally from index.scss we should do that.